Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -86,7 +86,10 @@ extension Command {
             var args = basicArgs
             args += ["-MD", "-MT", "dependencies", "-MF", path.deps]
             args += ["-c", path.source, "-o", path.object]
-
+            // Append -std flag if file is cpp.
+            if path.source.isCpp {
+               args += ["-std=c++14"]
+            }
```

Comment:
Let's accept the compiler defaults to start with, and figure out how to handle language specific arguments as a separate PR.